cmake/projects/YAML_CPP/hunter.cmake

```diff
@@ -0,0 +1,21 @@
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+  PACKAGE_NAME
+  YAML_CPP
```

Comment:
The C++ namespace is `YAML` so I guess we can name it just `YAML` (or even `yaml`).